External/FEXCore/Source/Interface/Core/OpcodeDispatcher/Vector.cpp

```diff
@@ -4620,23 +4620,52 @@ void OpDispatchBuilder::PCMPXSTRXOpImpl(OpcodeArgs, bool IsExplicit) {
   } else {
     IntermediateResult = _VPCMPISTRX(Src1, Src2, Control);
   }
-  OrderedNode *ResultNoFlags = _And(IntermediateResult, _Constant(0xFFFF));
 
-  // For the indexed variant of the instructions, if control[6] is set, then we
-  // store the index of the most significant bit into ECX. If it's not set,
-  // then we store the least significant bit.
+  OrderedNode *ResultNoFlags = _And(IntermediateResult, _Constant(0xFFFF));
```

Comment:
Looks like in the IsMask path, this doesn't need to mask the results since it is reading the bits directly.

Reply:
Could also probably be a Bfe instead of an And with a constant btw.